RiscVPlatformPkg/Universal/Sec/RiscVOpenSbiPpi.c

```diff
@@ -0,0 +1,343 @@
+/** @file
+The module to produce OpenSBI PPI.
+
+Copyright (c) 2020, Hewlett Packard Development LP. All rights reserved.<BR>
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Library/PeiServicesLib.h>
+#include <Ppi/RiscVOpenSbiPpi.h>
+#include <sbi/riscv_asm.h>
+#include <sbi/sbi_hart.h>
+#include <sbi/sbi_types.h>
+#include <sbi/sbi_platform.h>
+#include <sbi/sbi_init.h>
+
```

Comment:
we need function header for each function.

Reply:
Yup, I already wrote them and put them in the header. I'll check them and copy them here.